core/tests/protractor_utils/ProfilePage.js

```diff
@@ -73,6 +73,35 @@ var ProfilePage = function() {
   this.expectUserToNotHaveInterestPlaceholder = function() {
     expect(interestPlaceholder.isPresent()).toBe(false);
   };
+
+  this.expectToHaveExplorationCards = function() {
+    var allExplorationCardElements = element.all(
+      by.css('.protractor-test-exploration-dashboard-card'));
+    allExplorationCardElements.then(function(cards) {
+      if (cards.length === 0) {
+        throw 'There is no exploration card on this profile';
+      }
+      expect(cards.length).toBeGreaterThan(0);
+    });
+  };
+
+  this.expectToHaveExplorationCardByName = function(explorationName) {
+    var allExplorationCardElements = element.all(
+      by.css('.protractor-test-exploration-dashboard-card'));
+
+    var hasExplorationCard = allExplorationCardElements.filter(function(card) {
+      return card.element(
+        by.css('.protractor-test-exp-summary-tile-title')).
```

Comment:
Can we move this to the line above? 
Also maybe define this element on top.

Reply:
I'm not quite sure if I understood what you meant, but I refactored based on the reviews, and if I did something wrong in this line, please just tell me